### ISSUE TITLE
"Permission denied" is common and should not have a scary message

### ISSUE THREAD

**Issue by Redsandro** [pauldreik/rdfind#71]
I was running `rdfind` on a backup drive. I got hundreds of these scary messages:

> Dirlist.cc::handlepossiblefile: This should never happen. FIXME! details on the next row:

Comments in the code suggests files were possibly open and written to by someone else.

https://github.com/pauldreik/rdfind/blob/fa7752d6205e2071a758e7aa2d4e1c541aa96a6e/Dirlist.cc#L162-L169

Turns out those dirs were simply owned by other users. Workaround: use `sudo`.

I suggest a permission/user check and a less alarming error message.

**Comment by pauldreik** [comment]
I agree this text is too alarming.

**Comment by valerio-bozzolan** [comment]
I agree. We may want to mark this (#71) as "closed", and then join conversation here, that seems the same reporting: #32
